src/backend/distributed/sql/citus--9.4-2--9.4-3.sql

```diff
@@ -0,0 +1,7 @@
+-- 9.4-2--9.4-3 was added later as a patch to improve master_update_table_statistics
+CREATE OR REPLACE FUNCTION master_update_table_statistics(relation regclass)
+RETURNS VOID
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_update_table_statistics$$;
```

Comment:
I guess we started using the `citus` prefix instead of `master` as of Citus 10, but since this is an internal function, it is probably okay to use `citus` prefix here

Reply:
That's right, but `master_update_table_statistics` doesn't exist as an internal function, the internal function `citus_update_table_statistics` was introduced with improvement of this functionality in the commit we originally want to backport https://github.com/citusdata/citus/commit/2f30614fe343103f2c90c804f7c6e4c9d1cbfc71 I wanted to stay consistent with the original commit therefore the choice of using the name `citus_update_table_statistics` for the internal function. As you said, it shouldn't be a problem since it's an internal function.